qemu/tests/rh_kernel_update.py

```diff
@@ -42,7 +42,7 @@ def get_brew_url(mnt_path, download_root):
         logging.debug("Brew URL is %s" % url)
         return url
 
-    def install_rpm(session, url, upgrade=False, nodeps=False, timeout=600):
+    def install_rpm(session, url, upgrade=False, nodeps=False, timeout=3600):
```

Comment:
it's not a good idea to update the default value, better to update as a configurable param.